### PR TITLE
322 look feel details

### PR DESCRIPTION
**Pull request by cdmo**
For #322 (note I added favicon on my own because why not right).

### CODE REVIEW

File: app/controllers/catalog_controller.rb [hunk 1]
```diff
@@ -39,7 +39,6 @@ class CatalogController < ApplicationController
     config.add_show_tools_partial(:citation)
 
     config.add_nav_action(:bookmark, partial: 'blacklight/nav/bookmark', if: :render_bookmarks_control?)
-    config.add_nav_action(:search_history, partial: 'blacklight/nav/search_history')
```
Comment (banukutlu) [inline app/controllers/catalog_controller.rb]: 💯 👍

File: app/views/layouts/blacklight/base.html.erb [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
     <title><%= render_page_title %></title>
     <%= opensearch_description_tag application_name, opensearch_catalog_url(format: 'xml') %>
-    <%= favicon_link_tag %>
+    <%= favicon_link_tag asset_pack_path('psulib_blacklight/images/icons/favicon.ico') %>
```
Comment (banukutlu) [inline app/views/layouts/blacklight/base.html.erb]: :cool:

File: app/views/layouts/blacklight/base.html.erb [hunk 3]
```diff
@@ -18,15 +18,15 @@
   <body class="<%= render_body_class %>">
     <%= render partial: 'shared/header_navbar' %>
 
-    <div id="main-container" class="<%= container_classes %>">
+    <div id="main-containerx" class="<%= container_classes %> mt-4">
```
Comment (banukutlu) [inline app/views/layouts/blacklight/base.html.erb]: @cdmo `main-containerx` is x typo or intentional?

Comment (cdmo) [reply app/views/layouts/blacklight/base.html.erb]: typo, oops :(

Thanks for reading carefully!!!

Comment (cdmo) [reply app/views/layouts/blacklight/base.html.erb]:
```suggestion
    <div id="main-container" class="<%= container_classes %> mt-4">
```

**Pull request merged**